styleguide/_themes/derek/scss/patterns/cta.scss

```diff
@@ -0,0 +1,68 @@
+.cta-container {
+  @include make-container-fluid;
+  padding: $half-padding-full;
+}
+
+.cta-row {
+  @include rs-make-row;
+  margin: 0 auto;
+}
+
+.cta-columnBody,
+.cta-columnTitle {
+  @include make-md-column(8);
+  @include make-md-column-push(2);
+  text-align: center;
+}
+
+.cta-columnButton {
+  @include make-sm-column(6);
+  @include make-sm-column-push(3);
+  @include make-md-column(4);
+  @include make-md-column-push(4);
+}
+
+.cta-columnIcon {
+  @include make-xs-column(2);
+  @include make-xs-column-push(5);
+  text-align: center;
+}
+
+.cta-columnIconImage {
+  text-align: center;
+}
+
+.cta-columnSecondary {
+  @include make-md-column(8);
+  @include make-md-column-push(2);
+  font-weight: $semi-bold;
+  text-align: center;
+}
+
+// Subpanels.
+.ctaSubpanel-columnButton {
+  @include make-sm-column(10);
+  @include make-sm-column-push(1);
+  @include make-md-column(10);
+  @include make-md-column-push(1);
+}
```

Comment:
As illustrated here, subpanel-specific behavior may require additional classes to be created. In the case of the CTAs, the buttons were extremely narrow inside subpanels so we had to expand the width.  CTAs also featured a rewrite to maintainable css, but I tried to keep all the other patterns intact as much as possible.